### ISSUE TITLE
Errors in last cell  

### ISSUE THREAD

**Issue by sahildeshpande** [Linaqruf/sd-notebook-collection#46]
Couldn’t launch the notebook

**Comment by Linaqruf** [comment]
Please explain further; I am not clairvoyant.

**Comment by sahildeshpande** [comment]
Error when trying to launch. Something wrong with the Lora argument. And when I removed that argument from the code, then just some errors I couldn’t understand

**Comment by Linaqruf** [comment]
any screenshot?

**Comment by sahildeshpande** [comment]
Yes sure. Let me try again and get back to you

**Comment by sahildeshpande** [comment]
Ok so this time I ran with update webui and update extension checked and it launched without issues
